ipinfo_db/client.py

```diff
@@ -145,4 +165,4 @@ def _get_data_field(self, ip, field):
 
     def _get_data_dictionary(self, ip, fields):
         data = self.db.get(ip)
-        return {k:data[key] for key in fields if key in data}
\ No newline at end of file
+        return {key:data[key] for key in fields if key in data}
```

Comment:
Are we sure this is a safe change? Was this just bugged before?

Reply:
Yes, it was broken previously. Tested it.